acra/configuring-maintaining/general-configuration/acra-keys/extract-client-id.md

```diff
@@ -0,0 +1,49 @@
+---
+title: extract-client-id
+bookCollapseSection: true
+weight: 10
+---
+
+# extract-client-id
+
+**`extract-client-id`** is `acra-keys` subcommand used for generating ClientID from TLS certificate.
+
+## Command line flags
+
+### General flags
+
+* `--tls_cert=<path>` ❗
+
+  Path to TLS certificate to use as client_id identifier.
+
+* `--tls_identifier_extractor_type={distinguished_name|serial_number}`
+
+  Decide which field of TLS certificate to use as ClientID.
+
+    * `distinguished_name` — certificate Distinguished Name (DN)
+    * `serial_number` — certificate serial number
+
+* `--print_json`
+
+  Use machine-readable JSON output.
+
+
+  ❗ - flags required to be specified.
+
+
+## Usage example
+
+**`extract-client-id`** subcommand could be used to generate ClientID to reference some keys generated by [`acra-keymaker`][**`export`** subcommand]({{< ref "/acra/configuring-maintaining/general-configuration/acra-keymaker" >}}) via `tls_cert` flag and empty `client_id`.
+
+```
+$ acra-keys extract-client-id --tls_cert=/etc/acra-demo/configs/acra-client1.crt --print_json
```

Comment:
would be great to add example of output without `--print_json` flag. it will simplify understanding what to expect and how it can be used in another case.